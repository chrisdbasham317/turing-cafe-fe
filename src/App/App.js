import React, { Component } from 'react';
import './App.css';

class App extends Component {
  constructor() {
    super();
    this.state = {
      isLoading: true,
      reservations: []
    }
  }

  componentDidMount = () => {
    fetch('http://localhost:3001/api/v1/reservations')
      .then(res => res.json())
      .then(data => {
        this.setState({
          isLoading: false,
          reservations: data
        })
      })
  }

  render() {
    const resData = !this.state.isLoading ? 
       console.log(this.state.reservations) : console.log('not yet')
    return (
      <div className="App">
        <h1 className='app-title'>Turing Cafe Reservations</h1>
        <div className='resy-form'>

        </div>
        <div className='resy-container'>
          
        </div>
      </div>
    )
  }
}

export default App;
