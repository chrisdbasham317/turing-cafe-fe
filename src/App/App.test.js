import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import App from './App';

describe('App', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallow(<App />)
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})
