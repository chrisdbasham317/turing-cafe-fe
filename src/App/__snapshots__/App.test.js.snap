// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match the snapshot 1`] = `
<div
  className="App"
>
  <h1
    className="app-title"
  >
    Turing Cafe Reservations
  </h1>
  <Form
    addReservation={[Function]}
  />
  <div
    className="resy-form"
  />
  <div
    className="resy-container"
  />
</div>
`;
